spring-session/src/main/java/org/springframework/session/MapSession.java

```diff
@@ -40,7 +40,7 @@
  * @since 1.0
  * @author Rob Winch
  */
-public final class MapSession implements ExpiringSession, Serializable {
+public class MapSession implements ExpiringSession, Serializable {
```

Comment:
I removed the final because it is implemented with covering all required logic, which means we can use it for example for storing sessions in Mongo or relational databases by simply creating annotated class that extends the MapSession. Is there any particular reason for keeping it final?